.github/workflows/build_and_test.yml

```diff
@@ -11,6 +11,7 @@ on:
   schedule:
     # 03:00 every Saturday morning
     - cron: '0 3 * * 6'
+  workflow_dispatch:
```

Comment:
A lot of the steps below are triggered depending on `github.event_name`. You probably want to run the same checks as if it were a pull request, so I'd suggest looking for `github.event_name == 'pull_request'` and adding `workflow_dispatch` as an alternative trigger. (Similarly in some of the other workflows.)

Reply:
Do I need to do the "Check tket version bump" step for a manual trigger? As long as "Upload package" doesn't run, this isn't needed, right?

Reply:
A package is never uploaded on a pull-request workflow run, so I guess it's not necessary to do this check.